lib/macho/fat_file.rb

```diff
@@ -133,12 +133,15 @@ def dylib_id
 
     # Changes the file's dylib ID to `new_id`. If the file is not a dylib, does nothing.
     # @example
-    #  file.dylib_id = 'libFoo.dylib'
+    #  file.change_dylib_id('libFoo.dylib')
     # @param new_id [String] the new dylib ID
+    # @param options [Hash]
+    # @option options [Boolean] :strict (true) whether or not to fail loudly
+    #  with an exception if the change cannot be performed
```

Comment:
I think the wording of this documentation string needs to be tweaked a bit to be more specific and to better match the changed behavior we agreed on. (And of course this needs to change in all the other places this is used, too.)